sync/src/tasks/tests_dag.rs

```diff
@@ -87,83 +84,29 @@ async fn test_sync_dag_blocks() -> Result<()> {
         .await
         .expect("failed to init system");
 
-    test_system
-        .target_node
-        .chain()
-        .dag()
-        .save_dag_state(*G_TEST_DAG_FORK_STATE_KEY, DagState { tips: vec![] })?;
-    test_system
-        .local_node
-        .chain()
-        .dag()
-        .save_dag_state(*G_TEST_DAG_FORK_STATE_KEY, DagState { tips: vec![] })?;
+    let count = 10;
 
     let mut target_node = Arc::new(test_system.target_node);
     let local_node = Arc::new(test_system.local_node);
     Arc::get_mut(&mut target_node)
         .unwrap()
-        .produce_block(G_TEST_DAG_FORK_HEIGHT)
+        .produce_block(count)
         .expect("failed to produce block");
-    let dag_genesis_header = target_node.chain().status().head;
+    let dag_genesis_header_id = target_node.chain().get_block_dag_genesis()?;
```

Comment:
Please do a check on dag genesis between local node and target node.

Reply:
See the latest push.